Recrutify/Recrutify.DataAccess/Models/ParticipantSlot.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Recrutify.DataAccess.Models
+{
+    public class ParticipantSlot
+    {
+        public IEnumerable<int> BestTimeToConnectCanditate { get; set; }
```

Comment:
remove word Candidate from all namings of fields

Reply:
done